include/gf_vect_mul.h

```diff
@@ -64,8 +63,7 @@ extern "C" {
  * @returns 0 pass, other fail
  */
 
-int gf_vect_mul_sse(int len, unsigned char *gftbl, void *src, void *dest);
-
+	int gf_vect_mul_sse(int len, unsigned char *gftbl, void *src, void *dest);
```

Comment:
Don't need to run indent on external headers.  The extern "C" { braces mess it up.